fiftyone/utils/mapanything.py

```
@@ -0,0 +1,245 @@
+
+        model = mapanything_models.MapAnything.from_pretrained(
+            config.hf_repo
+        ).to(device)
+        model.eval()
+        return model
+
+    @property
+    def media_type(self):
+        return "image"
+
+    def _to_pil(self, img):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_to_pil`

(ANN202)